src/test/java/com/intel/gkl/testing_utils/TestingUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.intel.gkl.testing_utils;
+
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+public class TestingUtils {
+    public static final int SIZE_4MB = 4 *1024 * 1024;
+    public static final int MAX_ARRAY_SIZE = Integer.MAX_VALUE -2;
+
+    static SecureRandom rng = new SecureRandom();
+
+    public static void randomDNA(byte[] array) {
+        final byte[] DNA_CHARS = {'A', 'C', 'G', 'T'};
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] = DNA_CHARS[rng.nextInt(DNA_CHARS.length)];
+        }
+    }
+
+    public static byte[] generateRandomDNAArray(int size){
+        byte[] dna = new byte[size];
+        randomDNA(dna);
+        return dna;
+    }
+    public static byte[] generateSimpleDNAArray(int size) {
+        byte[] dna = new byte[size];
+        Arrays.fill(dna, (byte) 'A');
+        return dna;
+    }
+    public static int getMaxInputSizeForGivenOutputSize(int outputBufferSize, int level, boolean nowrap){
+        if(level != 0){
+            return outputBufferSize;
+        }
+        return (int) (0.99 * outputBufferSize);
```

Comment:
Why 0.99 here?

Reply:
Compression level set to 0 means no compression. But it still creates a little overhead just because of the compression format. I couldn't find a working formula for determining an exact overhead value, so it's an estimate that cuts a few last elements. It had to be done as the test expect that compressed data will fit into a single array that cannot be bigger than `MAX_ARRAY_SIZE`.

Reply:
Ok lets add a quick comment in the code for next time.